vtStor/Drive.cpp

```diff
@@ -17,21 +17,20 @@ limitations under the License.
 */
 #include <assert.h>
 
+#include "DeviceHandle.h"
 #include "Drive.h"
 
 namespace vtStor
 {
 
-cDrive::cDrive(std::shared_ptr<String> DevicePath) : m_DevicePath(DevicePath)
+cDrive::cDrive(std::shared_ptr<vtStor::cDeviceInterface> Device) :
```

Comment:
Update this constructor to also take in a device handle, so that the drive object doesn't have to worry about this process.